ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py

```diff
@@ -97,7 +97,7 @@ def process(self, device, results, log):
             pass
 
         # if NTDS service present then this is a DC
-        if domainController:
+        if len(domainController):
```

Comment:
if there are no results returned, domainController results in an empty list, []
so len([]) == 0

i made another change to make this less confusing